<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>window</class>
 <widget class="QMainWindow" name="window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>My Gui</string>
  </property>
  <widget class="QWidget" name="central">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QFrame" name="tableFrame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTableView" name="table">
          <property name="verticalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <property name="horizontalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="tableButtons" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="nCb">
             <property name="text">
              <string>close</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="nAb">
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Add row for typing name&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="text">
              <string>open</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="nSb">
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Write to file&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="text">
              <string>Write</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="nRb">
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Read from file&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="text">
              <string>Refresh</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QFrame" name="decoderFrame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QWidget" name="widget" native="true">
          <layout class="QGridLayout" name="gridLayout">
           <item row="1" column="1">
            <widget class="QPlainTextEdit" name="editor">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QPlainTextEdit" name="text">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="editor_label">
             <property name="text">
              <string>Editor</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="text_label">
             <property name="text">
              <string>Output</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="decoderButtons" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QCheckBox" name="editMod">
             <property name="text">
              <string>Edite</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="getBtn">
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Get text from Table's selected item&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="text">
              <string>Get</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="setBtn">
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Set editor's text to Table's selected item&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="text">
              <string>Set</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="swapBtn">
             <property name="text">
              <string>swap</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="autodecBtn">
             <property name="text">
              <string>auto code</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="decBtn">
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Decode text within editor by decoder&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="text">
              <string>Decode</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label2">
             <property name="text">
              <string>Encoder</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="encEdt"/>
           </item>
           <item>
            <widget class="QLabel" name="label1">
             <property name="text">
              <string>Decoder</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="decEdt">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_all">
   <property name="text">
    <string>Save all</string>
   </property>
  </action>
  <action name="actionReread">
   <property name="text">
    <string>Reread</string>
   </property>
  </action>
  <action name="actionReread_all">
   <property name="text">
    <string>Reread all</string>
   </property>
  </action>
  <action name="actionClose_2">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionClose_all">
   <property name="text">
    <string>Close all</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
